frlzjosh/project_management_system
# PMS-6: Key is generated, composer works, I have all libraries and packages to have your project serve on localhost

## Ready/Hold
`Is this ready to merge?` 
Yes
## Description
`What does this branch do?`
Generate key, has all needed packages
## Steps for Testing
`How do we check if this works?`
turn on wamp/mamp
type in localhost/project_management_system/public/
lemme know if it works.
